svg2vd.rb

```diff
@@ -0,0 +1,30 @@
+class Svg2vd < Formula
+  desc "A tool to convert Scalable Vector Graphics to Android Vector Drawables"
+  homepage "https://github.com/Shopify/svg2vd"
+  url "https://github.com/Shopify/svg2vd/releases/download/0.1/svg2vd-0.1.jar"
+  sha256 "5b39038edf01bb424d1b16a33e29bb439fccf1ba87153eeeb9de0a486bd87471"
+  version "0.1"
+
+  depends_on "openjdk@11"
```

Comment:
Depending on this doesn't set `JAVA_HOME` or anything so you're not left in a working state as is. It still executes the macOS wrapper and you're presented with ```The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.```

Given we internally use JDK bundled with Android Studio, externally it already provides helpful output, I think maybe dropping this dependency is OK.

Reply:
Makes sense to me. 👍